projects/igniteui-angular/src/lib/directives/for-of/base.helper.component.ts

```diff
@@ -21,17 +27,34 @@ export class VirtualHelperBaseDirective implements OnDestroy, AfterViewInit {
 
     private _afterViewInit = false;
     private _scrollNativeSize: number;
+    private _observer: ResizeObserver;
+    private _detached = false;
+    protected resizeNotify = new Subject();
+    protected destroy$ = new Subject<any>();
+
 
     ngAfterViewInit() {
         this._afterViewInit = true;
+        const destructor = takeUntil<any>(this.destroy$);
+        this.resizeNotify.pipe(destructor,
+        debounceTime(40, undefined))
```

Comment:
Let's debounce only on IE11 so that the more used browsers don't get the slight delay in restoring the scroll state